docs/_plugins/include_absolute.rb

```diff
@@ -0,0 +1,168 @@
+module Jekyll
```

Comment:
Is this module taken from somewhere or is it our doing?

Reply:
It's taken